workflow/cwl/helloworld-job.yml

```diff
@@ -0,0 +1,2 @@
+name: John Doe
```

Comment:
Note that there is no more `name` CLI argument; rather an `inputfile` containing the list of names to greet.  The current example call is:

```console
$ python code/helloworld.py \
       --inputfile inputs/names.txt
       --outputfile outputs/greetings.txt
       --sleeptime 0
```